Makefile

```diff
@@ -17,10 +16,12 @@ help: ## List of commands
 		awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
 	@echo "\nUsage: make <command>"
 
-build: ## Build both API and frontend
-	cd api && $(GO_BUILD) -o $(BINARY_NAME) .
+build: build-api ## Build both API and frontend
```

Comment:
We can maybe also add the instructions to build and run api, etc.. in the readme